docs/sdk/src/reference_docs/frame_currency.rs

```diff
@@ -1,8 +1,99 @@
 //! FRAME Currency Abstractions and Traits
 //!
-//! Notes:
+//! TODO:
 //!
-//! - History, `Currency` trait.
-//! - `Hold` and `Freeze` with diagram.
-//! - `HoldReason` and `FreezeReason`
-//! 	- This footgun: https://github.com/paritytech/polkadot-sdk/pull/1900#discussion_r1363783609
+//! - [x] History, `Currency` trait.
+//! - [ ] `Hold` and `Freeze` with diagram.
+//! - [x] `HoldReason` and `FreezeReason`
```

Comment:
These are specifically related to fungible, should be documented in the fungible docs?